### ISSUE TITLE
TypeError: options.ignore.map is not a function

### ISSUE THREAD

**Issue by joeneldeasis** [cdimascio/generator-express-no-stress#17]
Encountered this error below when trying to generate a new project

Operating System: Windows 10 Version 1709 (OS Build16299.371)
Node: v9.11.1
NPM: 5.6.0

```
X:\Codes>yo express-no-stress myapp
? App description [My cool app] My cool app
? API Root [/api/v1] /api/v1
? Version [1.0.0] 1.0.0
events.js:165
      throw er; // Unhandled 'error' event
      ^

TypeError: options.ignore.map is not a function
    at Object.generate (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\fast-glob\out\managers\tasks.js:9:37)
    at getWorks (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\fast-glob\out\index.js:49:29)
    at Function.sync (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\fast-glob\out\index.js:14:17)
    at tasks.reduce (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\globby\index.js:117:51)
    at Array.reduce (<anonymous>)
    at Function.module.exports.sync (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\globby\index.js:116:15)
    at EditionInterface.exports.copy (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\mem-fs-editor\lib\actions\copy.js:23:26)
    at module.exports.appStaticFiles (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\app\index.js:92:17)
    at Object.<anonymous> (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\yeoman-generator\lib\index.js:399:25)
    at C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\run-async\index.js:25:25
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\xxxx\AppData\Roaming\npm\node_modules\generator-express-no-stress\node_modules\yeoman-generator\lib\index.js:406:18)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5))
```

**Comment by chrismessiah** [comment]
@joeneldeasis did you manage to solve this? I also encounter this on

macOS 10.13
Node v9.11.1
npm 5.6.0

**Comment by cdimascio** [comment]
thanks. i just reprod it on mac. will have a look. thanks for the @chrismessiah and @joeneldeasis. im on it now

**Comment by cdimascio** [comment]
@joeneldeasis @chrismessiah This is resolved in version 3.5.2

**Comment by cdimascio** [comment]
i tested the fix with on mac with a variety of node versions including node v9.11.1 and npm v5.6.0.
closing this out.
please reopen if you still have issues
